meilisearch-release-changelog/v1.14.0.md

```diff
@@ -0,0 +1,137 @@
+
+
+Done by @ManyTheFish in [#5254](https://github.com/meilisearch/meilisearch/pull/5254).
+
+## Composite Embedders
+
+
+This feature allows using different embedders at search and indexing time, which is useful to optimize the embedder to each situation:
+
+- Using a remote embedder for indexing in bulk, as remote embedders provide the highest bandwidth (embeddings/s)
+- Using a local embedder for answering search queries, as local embedders provide the lowest latency (time to first embedding)
+
+To use the feature, follow these steps:
+
+1. Enable the `Composite embedders` feature from the Cloud dashboard or with the following:
+```
+curl -H 'Content-Type: application/json' MEILISEARCH_URL/experimental-features -d '{"compositeEmbedders": true}'
+```
+2. Send a settings task (`PATCH /indexes/INDEX_UID/settings`) containing an embedder with `source: "composite"`, and with the parameters `searchEmbedder` and `indexingEmbedder` describing the embedder at search time (resp. indexing time). For example, using a [Hugging Face inference endpoint](https://huggingface.co/inference-endpoints/dedicated)
+```jsonc
+{
+    "embedders": {
+        "text": {
+            "source": "composite",
+            "searchEmbedder": {
+                "source": "huggingFace", // locally computed embeddings using a model from the Hugging Face Hub
+                "model": "baai/bge-base-en-v1.5",
+                "revision": "a5beb1e3e68b9ab74eb54cfd186867f64f240e1a"
+            },
+            "indexingEmbedder": {
+                "source": "rest", // remotely computed embeddings using Hugging Face inference endpoints
+                "url": "https://URL.endpoints.huggingface.cloud",
+                "apiKey": "hf_XXXXXXX",
+                "documentTemplate": "Your {{doc.template}}",
+                "request": {
+                    "inputs": [
+                        "{{text}}",
+                        "{{..}}"
+                    ]
+                },
+                "response": [
+                    "{{embedding}}",
+                    "{{..}}"
+                ]
+            }
+        }
+    }
+}
+```
+3. Send documents to the index. They will be indexed remotely using the Hugging Face inference endpoint.
+4. Perform semantic search queries, they will be embedded using the local model fetched from the Hugging Face Hub.
+
+For more details about this feature, please refer to its [public usage page](https://www.notion.so/meilisearch/Composite-embedder-usage-14a4b06b651f81859dc3df21e8cd02a0?pvs=4#19f4b06b651f809ab3ecc0daae7d5856)
+<TBD: Find a better title>
+
+Done by @dureuill in [#5371](https://github.com/meilisearch/meilisearch/pull/5371).
+
+## Other improvements
+
+* Get multiple documents by ids by @dureuill in [#5384](https://github.com/meilisearch/meilisearch/pull/5384)
```

Comment:
Maybe a short dedicated section for this? It was requested by a lot of users, and you mentioned it in the introduction